hera_cal/tests/test_redcal.py

```diff
@@ -609,8 +577,8 @@ def test_firstcal(self):
             d[(ant1, ant2, pol)] *= fc_gains[(ant1, split_pol(pol)[0])] * np.conj(fc_gains[(ant2, split_pol(pol)[1])])
         for ant in gains.keys():
             gains[ant] *= fc_gains[ant]
-        meta, g_fc = rc.firstcal(d, freqs, conv_crit=0)
-        np.testing.assert_array_almost_equal(np.linalg.norm([g_fc[ant] - gains[ant] for ant in g_fc]), 0, decimal=3)
+        meta, g_fc = rc.firstcal(d, freqs)
```

Comment:
I'd suggest changing `g_fc` to `sol_fc` or similar to clarify what it actually is.